src/EllipsisNotation.jl

```diff
@@ -3,16 +3,45 @@ __precompile__()
 module EllipsisNotation
 
   import Base.getindex, Base.setindex!
-  const ..   = Val{:..}
+  const .. = Val{:..}()
+  const IndexType = Union{(VERSION < v"0.6-" ? Real : Int), Val{:..}}
```

Comment:
In my implementation, the intention of `(VERSION < v"0.6-" ? Real : Int)` is to avoid an error like `getindex(::Array{Int64,x}, ::Int64, …) is ambiguous` in v0.5.x. see -> https://gist.github.com/antimon2/b325690e0e530a9fd880bfc3afd48153#file-myellipsisgetindex-ipynb